docker/docker-init.sh

```diff
@@ -2,6 +2,11 @@
 
 set -ex
 
+  if [ $IS_KERBEROS_ENABLED ]; then
```

Comment:
How is this variable is being set?

Reply:
this was set as environment variable in deployment file and default value is false